Stream_support/include/CGAL/IO/binary_file_io.h

```diff
@@ -20,45 +20,81 @@
 
 #include <CGAL/config.h>
 #include <iostream>
-#include <boost/cstdint.hpp>
+#include <cstdint>
 
 namespace CGAL {
 
 inline void
-I_Binary_write_integer32(std::ostream& out, boost::int32_t i) {
+I_Binary_write_uinteger32(std::ostream& out, std::uint32_t u) {
+    out.write( (char*)(&u), 4);
+}
+// Special function to write size_t in 32b integer to ensure files
+// written by 64b systems are still readable by 32b ones
+inline void
+I_Binary_write_size_t(std::ostream& out, std::size_t s) {
```

Comment:
I think the name of the function should reflect that the value is supposed to be less than 4Gi (2^32).

Reply:
Good point. I added `into/from_uinteger32` to both functions.